chapters/en/slides/module3/module3_01.Rmd

```diff
@@ -0,0 +1,274 @@
+---
+params:
+  dynamictitle: "module3_01"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+# Visualizing data from one column
+
+Notes:
+In the previous modules we have seen how graphical marks in Altair
+to create plots
+and learn about which visual channels are effective
+for displaying data.
+Now that you are equipped with this knowledge and can create your own plots,
+we will dive deeper into when it is appropriate to use certain charts instead of others.
+
+---
+
+## The movies dataset
+
+```{python}
+import pandas as pd
+
+
+movies = pd.read_csv('movies.csv')
+movies
+```
+
+Notes:
+For this module,
+we will study a dataset of movies
+containing each movie's
+title, runtime, budget, revenue, genre, country of production, and average user rating.
+
+Upon see this table,
+there are several questions that might come to mind, e.g.
+
+- Are high budget movies more profitable?
+- Do some countries make more movies of certain genres?
+- Are some genres rated higher than others?
+
+Before diving into these comparisons,
+we want to understand the data for each of the columns in our dataset.
+
+Let's start by looking at the ratings the movies received
+and explore the best ways to visualize this data.
+
+---
+
+## Visualizing a single column with a point plot along one axis
```

Comment:
This is spelled incorrect on the rendered slides deployed by netlify, there it reads: "ReprestnData points from a single variable", not sure why the mismatch...

Reply:
It probably deployed an earlier commit and didn't update